### ISSUE TITLE
Teardown

### ISSUE THREAD

**Issue by RogerHardiman** [mbullington/yellowstone#9]
I wanted to test Yellowstone performing a teardown.
I added code to wowzs.js to catch ^C and call Close()

```
// Issue a Teardown when the process exits
process.on('SIGINT', function (){
  console.log('Send Teardown');
  client.close(true);
  process.exit(0);
});
```

But I get an error in the Yellowstone library

> yellowstone/lib/index.js:358
>       return new Promise(promise);
>                          ^
> 
> ReferenceError: promise is not defined

**Comment by mbullington** [comment]
Hm, okay, probably a small issue. I'll make sure to look into this and Tix timely, thanks!

**Comment by mbullington** [comment]
Fixed! Also, your other contributions are now included on npm in version v2.1.0, again thank you for contributing!

Please let me know if you what you're able to make with yellowstone (if you choose to use it for anything), and how I can help to make the library work better for it.
